### PR TITLE
doc/radosgw: add gateway starting command

### PR DESCRIPTION
**Pull request by zdover23**
Add a command that properly starts (or restarts) the RADOS gateway after RGW settings have been changed. This commit has been added in response to an issue reported anonymously on
https://pad.ceph.com/p/Report_Documentation_Bugs.

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "quincy"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

- When filling out the below checklist, you may click boxes directly in the GitHub web UI.  When entering or editing the entire PR message in the GitHub web UI editor, you may also select a checklist item by adding an `x` between the brackets: `[x]`.  Spaces and capitalization matter when checking off items this way.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
- `jenkins test rook e2e`
</details>

### CODE REVIEW

File: doc/radosgw/multisite.rst [hunk 1]
```diff
@@ -528,6 +528,15 @@ running the following commands on the object gateway host:
    systemctl start ceph-radosgw@rgw.`hostname -s`
    systemctl enable ceph-radosgw@rgw.`hostname -s`
 
+If you have used the ``radosgw-admin`` command, you will not be able to use
```
Comment (anthonyeleven) [inline doc/radosgw/multisite.rst:531]: I'm skeptical of this wording.  The above steps include a pool deletion that is qualified as only to be run if there is no data, implying that the other steps are be run regardless.  I don't understand now `radosgw-admin` implies that there are no services.  I suggest that this be run by the RGW leads.

Oh and above the mention of `civetweb` is outdated.

Comment (cbodley) [reply doc/radosgw/multisite.rst:531]: i think this is pointing at the difference between package vs containerized deployments, but confuses `radosgw-admin` for `cephadm`

File: doc/radosgw/multisite.rst [hunk 2]
```diff
@@ -528,6 +528,16 @@ running the following commands on the object gateway host:
    systemctl start ceph-radosgw@rgw.`hostname -s`
    systemctl enable ceph-radosgw@rgw.`hostname -s`
 
+If you have used the ``cephadm`` command, you will not be able to use
```
Comment (anthonyeleven) [inline doc/radosgw/multisite.rst:531]: If this cluster was deployed with ``cephadm``,

**Review by anthonyeleven — APPROVED** [review]

File: doc/radosgw/multisite.rst (hunk 2 above)
Comment (zdover23) [reply doc/radosgw/multisite.rst:531]: Accepted with slight modification to reduce ambiguity.

**Pull request merged**